### PR TITLE
A few issue template improvements

### PR DESCRIPTION
**Pull request by stephencelis**
Just a few small improvements.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,23 +8,26 @@ assignees: ''
 ---
 
 **Describe the bug**
-A clear and concise description of what the bug is.
+Give a clear and concise description of what the bug is.
```
Comment (stephencelis) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Making the voice consistent.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -8,23 +8,26 @@ assignees: ''
...
 
 **To Reproduce**
+Zip up a project that reproduces the behavior and attach it by dragging it here.
```
Comment (stephencelis) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: Think it's good to let folks know that GitHub supports zipped attachments.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -8,23 +8,26 @@ assignees: ''
...
 
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
 **Environment**
- - OS: [e.g. iOS]
- - Version [e.g. 13]
+ - Xcode [e.g. 11.4.1]
+ - Swift [e.g. 5.2.2]
+ - OS (if applicable): [e.g. iOS 13]
```
Comment (stephencelis) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: Feel like these are more applicable to a library.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mbrandonw) [reply .github/ISSUE_TEMPLATE/bug_report.md:14]: oh nice! is that new?

Comment (stephencelis) [reply .github/ISSUE_TEMPLATE/bug_report.md:14]: Not sure!

**Pull request merged**